packages/gatsby/src/utils/babel-loader-helpers.ts

```diff
@@ -16,13 +29,23 @@ const loadCachedConfig = () => {
   return pluginBabelConfig
 }
 
-const getCustomOptions = stage => {
+export const getCustomOptions = (stage: string): IBabelStage["options"] => {
```

Comment:
There's a string enum `Stage` which should be used for the argument here. You'll find it in commands/types.ts